hypha/apply/funds/views.py

```diff
@@ -436,6 +437,11 @@ class SubmissionAdminListView(BaseAdminSubmissionsTable, DelegateableListView):
     ]
 
 
+@method_decorator(staff_required, name='dispatch')
+class GroupingApplicationsListView(TemplateView):
```

Comment:
@vimal1083, can you add a bit of doc string on what this view is doing, and how it is used.

As this doesn't have any context/view-code and only relies on the template; makes it hard to understand when looking individually.